### PR TITLE
Add personal information to YAML 

### PR DESCRIPTION
**Pull request by aimundo**
I want to add my information to the YAML file

### Description of the proposed additions or changes

Add personal information to YAML

### Related issues

I thought my info. was added last time, but seems it did not go through

### Suggested reviewers (optional)

@agitter 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -1372,3 +1372,22 @@ authors:
         order: 6
         contributions:
           - Writing - Review & Editing
+          
+          -
```
Comment (agitter) [inline content/metadata.yaml:1376]:
```suggestion
  - 
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -1372,3 +1372,22 @@ authors:
...
+          
+          - 
+    name: Ariel I. Mundo
+    affiliations:
+      - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
+    email: aimundo@uark.edu
+    orcid: 0000-0002-6014-4538
+    coi:
+      string: "TBD"
+      lastapproved:
```
Comment (agitter) [inline content/metadata.yaml:1384]:
```suggestion
      lastapproved:  !!str 2021-12-19
```

File: content/metadata.yaml [hunk 3]
```diff
@@ -1372,3 +1372,22 @@ authors:
...
           - Writing - Review & Editing
+          
+          - 
+    name: Ariel I. Mundo
+    affiliations:
+      - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
+    email: aimundo@uark.edu
+    orcid: 0000-0002-6014-4538
+    coi:
+      string: "TBD"
```
Comment (agitter) [inline content/metadata.yaml:1383]:
```suggestion
      string: "None"
```
Do you have any (potential) conflicts of interest?  If not, then we'll enter "None" here.

File: content/metadata.yaml [hunk 4]
```diff
@@ -1372,3 +1372,22 @@ authors:
...
+      lastapproved: 
+    manuscripts:
+      diagnostics:
+        order:
+        contributions:
+          - Writing - Review & Editing
+          
+          
+          
+
```
Comment (agitter) [inline content/metadata.yaml:1393]:
```suggestion
```

File: content/metadata.yaml [hunk 5]
```diff
@@ -1372,3 +1372,22 @@ authors:
         order: 6
         contributions:
           - Writing - Review & Editing
+          
+          - 
+    name: Ariel I. Mundo
```
Comment (agitter) [inline content/metadata.yaml:1377]:
```suggestion
    github: aimundo
    name: Ariel I. Mundo
```

File: content/metadata.yaml [hunk 6]
```diff
@@ -1372,3 +1372,22 @@ authors:
...
+    affiliations:
+      - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
+    email: aimundo@uark.edu
+    orcid: 0000-0002-6014-4538
+    coi:
+      string: "TBD"
+      lastapproved: 
+    manuscripts:
+      diagnostics:
+        order:
```
Comment (agitter) [inline content/metadata.yaml:1387]:
```suggestion
        order: 1
```
I don't know if this will work without a value.  Let's add a placeholder for now.

**Review by agitter — CHANGES_REQUESTED** [review]
Thanks for adding yourself.  I have one question about conflicts of interest.  The rest are the changes are yaml formatting.

File: content/metadata.yaml (hunk 3 above)
Comment (aimundo) [reply content/metadata.yaml:1383]: I have no conflicts of interest

**Review by agitter — APPROVED** [review]
Looks good to me
image

**Pull request merged**